Add gps rescue notes for 4.5

ctzsnooze opened the pull request:
First draft of GPS Resue notes for 4.5

Asizon commented on docs/wiki/archive/GPS-Rescue-v4-5.md:
```diff
@@ -0,0 +1,526 @@
+
+:::danger
+
+- Only use the 10.10 Configurator GUI. For earlier Configurator versions, only make changes to GPS Rescue parameters in the CLI!
```
I would change Configurador versión namimg to 10.10.x

ctzsnooze replied:
Thanks, will do

Asizon reviewed (changes requested):
Awesome writing like always,i have made one small sugestión if you want to change

amhyder commented on docs/wiki/archive/GPS-Rescue-v4-5.md:
```diff
@@ -0,0 +1,526 @@
+# GPS Rescue 4.5
+
+Betaflight 4.5 makes the following significant improvements to GPS rescue:
+- greatly improved support for UBlox GPS modules, particularly M9 and M10, improving rescue reliability and smoothness.  
```
Is this line worth capitalizing to make it adhere to the rest of the lines below? If so This line (4) as well as line 14 should be modified.

ctzsnooze replied:
good point, thank you

ctzsnooze commented:
Should be good to merge now.
Made a number of changes and fixes, the main ones being:
- updated the NMEA information, to match https://github.com/betaflight/betaflight/pull/13044
- updated the debug list, to match https://github.com/betaflight/betaflight/pull/13055 and other changes
- updated comments about RXLOSS to match https://github.com/betaflight/betaflight/pull/13033
